readme.md

```diff
@@ -44,6 +44,23 @@ console.log(await getPort({port: portNumbers(3000, 3100)}));
 // Will use any port from 3000 to 3100, otherwise fall back to a random port
 ```
 
+Use the `clearLockedPorts()` to clear the internal cache of locked ports.
```

Comment:
This should be placed with the `clearLockedPorts` API docs

Reply:
Sorry, I didn't quite understand. I was referring to the `portNumbers` method.